### PR TITLE
ADD: Content Filtering

### PR DESCRIPTION
**Pull request by sihrc**
Should merge Chris/resize-images first - fixes data length cap and passes tests

### CODE REVIEW

**Comment by sihrc** [comment]
@madisonmay @anneCarlson code review before merge?

File: R/content_filtering.R [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+#' @seealso \code{\link{face_emotion}}, \code{\link{facial_features}}
+#' @export
+#' @import httr rjson stringr
+#' @examples
+#' img_list = list()
+#' img_list[[1]] = matrix(runif(64*64, 0, 1), nrow = 64)
+#' probabilities <- batch_content_filtering(img_list)
+#'
+batch_content_filtering <- function(imgs, api_key = FALSE, cloud = FALSE, ...) {
+  img_list <- format_images(imgs, 64)
```
Comment (madisonmay) [inline R/content_filtering.R]: Resize to 128 rather than 64.

**Comment by anneCarlson** [comment]
This looks good to me, but take my R knowledge with several grains of salt.

**Comment by sihrc** [comment]
I just don't know what we're doing with image resizing. That could be the only thing left here

**Pull request merged**